udi-sqlite-extensions.c

```diff
@@ -16,7 +18,14 @@ int udi_sqlite_init_extensions(sqlite3 *db, char **pzErrMsg,
   rc = sqlite3_auto_extension((void (*)())sqlite3_ulid_init);
   if (rc != SQLITE_OK)
   {
-    fprintf(stderr, "❌ demo.c could not load sqlite3_ulid_init: %s\n", sqlite3_errmsg(db));
+    fprintf(stderr, "❌ udi-sqlite.c could not load sqlite3_ulid_init: %s\n", sqlite3_errmsg(db));
+    sqlite3_close(db);
+    return 1;
+  }
+  rc = sqlite3_auto_extension((void (*)())sqlite3_crypto_init);
+  if (rc != SQLITE_OK)
+  {
+    fprintf(stderr, "❌ udi-sqlite.c could not load sqlite3_crypto_init: %s\n", sqlite3_errmsg(db));
```

Comment:
This file is called `udi-sqlite-extensions.c` so match the name in the error message.